flow/scripts/odb2def.tcl

```diff
@@ -0,0 +1,8 @@
+source $::env(SCRIPTS_DIR)/load.tcl
+load_design 3_2_place_iop.odb 2_floorplan.sdc "Starting global placement"
+
+
+write_def $::env(RESULTS_DIR)/3_2_place_iop.def
+write_verilog $::env(RESULTS_DIR)/3_2_place_iop.v
+
+exit
```

Comment:
What is the difference between using https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/8d1989f94a95cfa39bd77c3b538a285689654215/flow/Makefile#L756-L757?

Reply:
I was not able to run this %.def target, so I added this.

Reply:
Calling directly does not work?
```
ODB_FILE=<PATH> DEF_FILE=<PATH> openroad write_def.tcl
```
If you want to keep your script, it would be nice to have it generic -- same as def2odb / odb2files